op-node/rollup/finalized/finalized.go

```diff
@@ -0,0 +1,38 @@
+package finalized
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/log"
+
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+)
+
+type finalized struct {
+	derive.L1Fetcher
+	l1Finalized func() eth.L1BlockRef
+}
+
+func NewFinalized(l1Finalized func() eth.L1BlockRef, fetcher derive.L1Fetcher) *finalized {
+	log.Warn("NewFinalized", "l1Finalized", l1Finalized)
```

Comment:
Does this need to be a warning here and below?

Reply:
Removed it, it was just leftover from debugging.